docs/api/indexers/01-endpoints.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: "endpoints"
+sidebar_position: 1
+title: "Endpoints"
+---
+
+import LinkPreview from "@site/src/components/LinkPreview";
+
+:::tip Chains: Individual vs. Aggregated
+
+While subgraphs have a chain-focused approach (one subgraph endpoint/deployment for every chain), it is important to
```

Comment:
Do you mean "one subgraph endpoint/deployment for every chain" or "one subgraph endpoint/deployment per chain"?